public/app/features/dashboard/components/SupportSnapshot/SupportSnapshot.tsx

```diff
@@ -255,12 +270,18 @@ export function SupportSnapshot({ panel, plugin, onClose }: Props) {
 
           <AutoSizer disableWidth>
             {({ height }) => (
-              <iframe
-                src={`/dashboard/new?orgId=${contextSrv.user.orgId}&kiosk`}
-                width="100%"
-                height={height - 100}
-                frameBorder="0"
-              />
+              <>
+                <iframe
+                  src={`/dashboard/new?orgId=${contextSrv.user.orgId}&kiosk`}
+                  width="100%"
+                  height={height - 100}
+                  frameBorder="0"
+                  style={{
+                    display: iframeLoading ? 'block' : 'none',
+                  }}
+                />
+                {!iframeLoading && <div>No iframe support?</div>}
+              </>
```

Comment:
think we should probably remove this iframe until we change the default x-frame-options from deny to sameorigin. I would expect the vast majority have embedding disabled. Cloud has it disabled (and I don't think you can change it without asking support).

Reply:
and if it is disabled the next dashboard you load after seeing this error message will be the snapshot :)